docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-merchant-portal-core-feature-integration.md

```diff
@@ -239,6 +266,16 @@ class MerchantUserDependencyProvider extends SprykerMerchantUserDependencyProvid
             new MerchantAclMerchantUserPostCreatePlugin(),
         ];
     }
+    
+    /**
+     * @return array<\Spryker\Zed\MerchantUserExtension\Dependency\Plugin\MerchantUserRoleFilterPreConditionPluginInterface>
+     */
+    protected function getMerchantUserRoleFilterPreConditionPlugins(): array
+    {
+        return [
+            new AclMerchantPortalMerchantUserRoleFilterPreConditionPlugin(),
```

Comment:
Please add verification steps for newly added plugin**s**.

Reply:
fixed